omod/src/main/webapp/pages/editSection.gsp

```diff
@@ -10,8 +10,11 @@
 
     ui.includeCss("registrationapp", "editSection.css")
 
-    def genderOptions = [ [label: ui.message("emr.gender.M"), value: 'M'],
-                          [label: ui.message("emr.gender.F"), value: 'F'] ]
+    def translatedGenderOptions = []
+                              
+    genderOptions.each { optn ->
+    	translatedGenderOptions << [label: ui.message("emr.gender." + optn), value: optn]
+    }
```

Comment:
This would not work right?
```groovy
genderOptions.each { optn ->
  genderOptions << [label: ui.message("emr.gender." + optn), value: optn]
}
```
Then just name it `localizedGenderOptions` or `l10nGenderOptions `.

Reply:
Thanks @mks-d, the above won't work. So I'm opting for `localizedGenderOptions`.